terraform/modules/agreements-db/main.tf

```diff
@@ -0,0 +1,23 @@
+#########################################################
+# Database: Agreements
+#
+# Provision Agreements Postgres Database.
+#########################################################
+data "cloudfoundry_service" "postgres" {
+  name = "postgres"
+}
+
+data "cloudfoundry_org" "ccs_conclave_cii" {
```

Comment:
Should this just be `data "cloudfoundry_org" "shared"` or similar (wasn't sure about the reference to Conclave here?)

Reply:
Ah - yep - good spot - quite right 👍